test/format-error/format-error.tf

```diff
@@ -9,5 +9,5 @@ terraform {
 }
 
 resource "random_id" "id" {
-    byte_length = 8
```

Comment:
This might be the reason why you had to change the format-error test to a success I think auto formatting broke the test. 

We should figure out if there is a way to ignore auto-format in this file.

Reply:
Yeah, exactly. The autoformat formatted it and therefore broke the test without me realizing it.